src/shared/cgroup-setup.c

```diff
@@ -17,6 +17,67 @@
 #include "fileio.h"
 #include "user-util.h"
 #include "fd-util.h"
+#include "virt.h"
+
+static bool cg_is_unified_possible(void) {
```

Comment:
i think it would be better if this function propagates errors properly (i.e. returns `int` rather than `bool`) but the single caller of it then eats up all errors. 

i.e. eating up errors is sometimes OK (like in this case), but we should minimize the code where we do that, hence try to do it once high up instead of many times down in the details

Reply:
ack, will do